nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/AbstractAzureDataLakeStorageProcessor.java

```diff
@@ -95,7 +95,7 @@
 
     public static final PropertyDescriptor DIRECTORY = new PropertyDescriptor.Builder()
             .name("directory-name").displayName("Directory Name")
-            .description("Name of the Azure Storage Directory. It will be created if not already existing")
+            .description("Name of the Azure Storage Directory. In case of the PutAzureDatalakeStorage processor, It will be created if not already existing")

```

Comment:
```suggestion
            .description("Name of the Azure Storage Directory. In case of the PutAzureDatalakeStorage processor, it will be created if not already existing")
```